File: ThirdYear/5thSemester/CN/Assignment-3/Assignment3/src/Question4.java
class MyRunnable implements Runnable {
    public void run() {
        System.out.println("Thread is running");
    }
}

public class Question4 {
    public static void main(String[] args) {
    	System.out.println("--------------------------------------------------------------");
    	System.out.println("Name : Ujjwal Kumar");
    	System.out.println("Registration Number : 2141011063");
    	System.out.println("Section  : C");
    	System.out.println("--------------------------------------------------------------");
        Thread myThread = new Thread(new MyRunnable());
        myThread.start();
    }
}

File: ThirdYear/5thSemester/PIP/Major Assignment/MajorAssignment-1/Question4.py
# Name : UJJWAL KUMAR
# Registration Number : 2141011063
# Section : C

# ----------------------- MAJOR ASSIGNMENT - 1 --------------------------------------

# 4. Write a function that generates a random password. The password should have a random length of
# between 7 and 10 characters. Each character should be randomly selected from positions 33 to 126
# in the ASCII table. Your function will not take any parameters. It will return the randomly generated
# password as its only result. Display the randomly generated password in your file’s main program.

import random
import string

def generateRandomPass():
    password = ''.join(random.choice(string.printable[33:127]) for i in range(random.randint(7,10)))
    return password

def main():
    print("The Password is ", generateRandomPass())

if __name__=='__main__':
    main()